src/cow-react/utils/amountFormat/README.md

```diff
@@ -0,0 +1,54 @@
+# Amounts formatting
+
+The application needs to display amounts of different sizes, from extremely small to extremely large, while maintaining accuracy.
+
+There are two types of amounts:
+
+- token amount: `100 000.54 COW`
+- fiat amounts: `≈ $1 946 628.4`
+
+### Feature-flags
+
+`localStorage.setItem('amountsRefactoring', '1')` - to highlight components with formatted tokens and fiat amounts
+
+### Language-sensitive number formatting
+
+The solution uses [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) API for numbers formatting.
+
+### Trailing zeros
+
+All functions and components mentioned in this documents are using `trimTrailingZeros()` function to avoid values with trailing zeros.
+
+Example: `trimTrailingZeros('12.092000') -> 12.092`
```

Comment:
Question that I still have on this.

What about `12.00000`? Will that turn into `12`, `12.` or `12.0`?

Reply:
Will be `12`, thanks! Added example into docs